src/JSONWriter.cs

```diff
@@ -47,7 +47,19 @@ static void AppendValue(StringBuilder stringBuilder, object item)
                         stringBuilder.Append(str[i]);
                 stringBuilder.Append('"');
             }
-            else if (type == typeof(byte) || type == typeof(int))
+            else if (type == typeof(byte) || type == typeof(sbyte))
```

Comment:
you could merge these || statements into a single line - since they all seem to be doing the same stringBuilder.Append(item.ToString())

Reply:
Yah I didn't because I found it a tad more readable to pair signed and unsigned in a single if rather than a gigantic if.